### PR TITLE
Add commit test

### PR DESCRIPTION
**Pull request by designdevelop**
ㅎㅎ

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -18,4 +18,4 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+SOFTWARE.
```
Comment (geoseong) [inline LICENSE:21]: what is this

Comment (geoseong) [inline LICENSE:21]: what is this???

Comment (designdevelop) [reply LICENSE:21]: it's just commit test

**Comment by geoseong** [comment]
@designdevelop please close this pull request

**Pull request closed without merging**